minikerberos/common.py

```diff
@@ -10,20 +10,24 @@
 import collections
 from minikerberos.constants import *
 from minikerberos.encryption import string_to_key, Enctype	
-from minikerberos.ccache import CCACHE	
-	
+from minikerberos.ccache import CCACHE
+from minikerberos.keytab import Keytab
+
+
 class KerberosSecretType(enum.Enum):
 	PASSWORD = 'PASSWORD'
 	PW = 'PW'
 	PASS = 'PASS'
 	NT = 'NT'
-	AES = 'AES'
```

Comment:
Please don't remove AES enum type, I see no need to separate it into two. In case you need it, okay but keep the AES as-is. Some packages depend on it already and not much gain comes with deleting it.

Reply:
I hit some problems when using only AES. Don't remember exactly what (I think the wrong length was used when decrypting). Had to separate them. I agree about the naming. My miss.

Reply:
I think I got it fixed now, but please drop an issue or a new PR if you run into problems.